pkgs/applications/networking/cluster/terraform/default.nix

```
@@ -62,21 +62,27 @@ let
 
           # Make providers available in Terraform 0.13 and 0.12 search paths.
           pluginDir = lib.concatMapStrings (pl: let
-            inherit (pl) repo version GOOS GOARCH;
-            inherit (pl.passthru) providerSourceAddress;
+            inherit (pl) version GOOS GOARCH;
+
+            pname = pl.pname or (throw "${pl.name} is missing a pname attribute");
+
+            # This is just the name, without the terraform-provider- prefix
+            plugin_name = lib.removePrefix "terraform-provider-" pname;
+
+            slug = pl.passthru.provider-source-address or "nixpkgs/${plugin_name}";
```

Host is missing the provider source address.
```suggestion
            slug = pl.passthru.provider-source-address or "registry.terraform.io/nixpkgs/${plugin_name}";
```